src/main/java/httpserver/HTTPServer.java

```diff
@@ -0,0 +1,22 @@
+package httpserver;
+
+public class HTTPServer {
+
+    private final SocketServer socketServer;
+    private final int port;
```

Comment:
I was thinking that you could use a short instead of a int.

After some research it's seems better to use the int. So my comment is just to share the link I find out about it!

http://programmers.stackexchange.com/questions/305704/should-udp-tcp-port-numbers-represented-as-a-short-or-integer